src/shadertools/modules/fp64/fp64-arithmetic.glsl.js

```diff
@@ -49,6 +49,12 @@ vec2 split(float a) {
   return vec2(a_hi, a_lo);
 }
 
+vec2 split2(vec2 a) {
```

Comment:
Would be nice with one-line comment on all of these helper methods, explain the basic idea of each.